File: Reverse Degree of a String/Reverse Degree of a String.py
# Solution for Reverse Degree of a String in PY

class Solution:
    def reverseDegree(self, s: str) -> int:
        ans = 0

        for i in range(1, len(s) + 1):
            ans += (26 - (ord(s[i - 1]) - ord('a'))) * i

        return ans

File: Reverse Degree of a String/Reverse Degree of a String.java
// Solution for Reverse Degree of a String in JAVA

class Solution {
    public int reverseDegree(String s) {
        int ans = 0;

        for (int i = 1; i <= s.length(); i++) {
            ans += (26 - (s.charAt(i - 1) - 'a')) * i;
        }

        return ans;
    }
}
